taskcluster/tc-asserts.sh

```diff
@@ -511,3 +511,22 @@ run_cpp_only_inference_tests()
   set -e
   assert_correct_ldc93s1_lm "${phrase_pbmodel_withlm_intermediate_decode}" "$status"
 }
+
+run_js_streaming_inference_tests()
```

Comment:
I suspect we will need that for more than JS, maybe there is no need to name it like that? Even if we add testing later

Reply:
As long as we keep it consistently as a "--stream" parameter it should be fine. I'll leave the renaming for whenever we add another binding. Python already has streaming API coverage through the concurrent streams test.